mvlearn/semi_supervised/ctclassifier.py

```diff
@@ -305,30 +305,30 @@ def fit(self, Xs, y):
                 it += 1
 
                 # fit each model to its respective view
-                self.estimator1_.fit(X1[L], y[L])
-                self.estimator2_.fit(X2[L], y[L])
+                self.estimator1.fit(X1[L], y[L])
+                self.estimator2.fit(X2[L], y[L])
```

Comment:
the fit should not change the init parameters but just add / modify attributes that end with _. So you should create 2 new attributes here called estimator1_ and estimator2_ (on top of estimator1 and estimator2)